src/main/resources/role.json

```diff
@@ -102,6 +102,12 @@
           "mandatory": true
         },
         "attributes": {
+          "jurisdiction": {
+            "mandatory": true
+          },
+          "caseType": {
+            "mandatory": true
+          },
```

Comment:
why you made jurisdiction and caseType are mandatory fields in incoming request. we are persisting these attributes only for query purpose. do we have any contract to make these attributes as a mandatory?

Reply:
This was suggested by Jon to make it mandatory. check his latest email.

Reply:
ok that's fine.